main.py

```diff
@@ -84,7 +84,10 @@ def remove_stop_words(frequency_list):
 	search_mode = False
 
 url = wikipedia_api_link + string_query
-response = requests.get(url)
+try:
+    response = requests.get(url)
+except Exception, e:
```

Comment:
I can shift the wole code in try statement. If you want an other way to verify if there was an exception, i'll work on it when i have the time! :)